### PR TITLE
Migrate CreateOrganization.js to styled components

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/3289

# Screenshots

image

### CODE REVIEW

File: components/CreateOrganization.js [hunk 1]
```diff
@@ -149,10 +120,10 @@ class CreateOrganization extends React.Component {
                   onChange={this.resetError}
                 />
 
-                <div className="result">
-                  <div className="success">{this.state.result.success}</div>
-                  <div className="error">{this.state.result.error}</div>
-                </div>
+                <Container textAlign="center" marginBottom="5rem">
+                  <Container style={{ color: 'green' }}>{this.state.result.success}</Container>
+                  <Container style={{ color: 'red' }}>{this.state.result.error}</Container>
```
Comment (Betree) [inline components/CreateOrganization.js:125]: It's better to use the `css` prop when you want to add styles to a styled component (see https://styled-components.com/docs/api#css-prop). However, the `Container` component has support for the `color` prop, and we can even use the design system directly from there.

```suggestion
                  <Container color="green.500">{this.state.result.success}</Container>
                  <Container color="red.500">{this.state.result.error}</Container>
```

**Review by Betree — COMMENTED** [review]
Good job, just a small improvement on the `color` prop but you nailed the rest :tada:

File: components/CreateOrganization.js (hunk 1 above)
Comment (SudharakaP) [reply components/CreateOrganization.js]: @Betree : Thanks for the suggestion. I've corrected this.

**Comment by Betree** [comment]
:tada:

**Pull request merged**